File: selection_sort/selection_sort.py
def searchMinor(array):
    minor = array[0] # first set a variable to find a minor number in array in this case we use the first element of the array
    minor_index = 0 # and the same for your index

    for i in range(1, len(array)): # so to 1 until the end of the array 
        if array[i] < minor: # check if the element is minor then we have
            minor = array[i] # if is we set a new minor
            minor_index = i # and same for the index

    return minor_index # return local of the minor number of the array

def selection_sort(array):
    sort_array = [] # create a new array to put the sort valeus. ASC

    for i in range(len(array)): # in the range of the array
        minor = searchMinor(array) # search a minor element
        sort_array.append(array.pop(minor)) # insert in the sort array end delete from the original array
    return sort_array # return the sorted array

print(selection_sort([5,3,6,2,10])) # Insert numbers here for sort!

File: selection_sort/selection_sort.js
// Function to find index with minor element in array
function find_minor(array) {
    let minor = array[0]; // set minor value as first element of array
    let minor_index = 0; // set index of the minor element

    for (let i = 1; i < array.length; i++){
        if (array[i] < minor) {
            minor = array[i];
            minor_index = i;
        }
    }
    return minor_index; // return index of minor value
}

function selection_sort(array){
    let sorted_array = []; // New array for save sorted values

    while (array.length > 0) { // while array is not empty
        let minor_index = find_minor(array); // find index of minor element
        let minor_value = array.splice(minor_index, 1)[0]; // remove minor value and return then
        sorted_array.push(minor_value); // add minor element to sorted array
    }
    return sorted_array; // return sorted_array
}

let array = [7, 3, 5, 2, 9];
console.log(selection_sort(array));